Documentation/cmdref/cilium-agent.md

```diff
@@ -54,6 +54,7 @@ cilium-agent [flags]
       --enable-policy string                       Enable policy enforcement (default "default")
       --enable-tracing                             Enable tracing while determining policy (debugging)
       --encrypt-interface string                   Transparent encryption interface
+      --encrypt-node                               Encrypt node IP
```

Comment:
"Encrypt node IP" seems too abstract. Can you write it differently?

Reply:
Sure all though I had a bit of trouble with the one-liner explanation,

"Enables encrypting traffic from non-Cilium pods and host networking"

Reply:
@jrfastab yep that sounds good!